msal/tests/Test.MSAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -42,6 +42,19 @@ class XamarinMSALDroidTests
         Platform platform;
         ITestController xamarinController;
 
+        UserQueryParameters DefaultUser
```

Comment:
commented elsewhere. this thing can be in a helper .... and potentially the other user types can as well...